.travis.yml

```diff
@@ -36,6 +36,7 @@ script:
    - ./autogen.sh
    - ./configure
    - make
+   - ./test-tool/iscsi-test-cu -d -A -V --test=iSCSI.iSCSITMF iscsi://127.0.0.1:3260/${TARGET}/0
```

Comment:
$git remote -v show
origin	https://github.com/gostor/libiscsi.git (fetch)
origin	https://github.com/gostor/libiscsi.git (push)

$ ./test-tool/iscsi-test-cu -l | grep iSCSI.iSCSITMF

Should be the same repo travis is using.